src/server/index.js

```diff
@@ -13,9 +13,34 @@ app.use(bodyParser.urlencoded({extended: true}));
 app.use(express.static(__dirname + '/../../dist'));
 app.use(cors());
 
-app.get('/api/:menuType', function(req, res) {
-  res.status(200);
-  MenuList.fetch(req.params.menuType, res.send.bind(res));
+app.post('/menu/:restaurantId', function(req, res) {
```

Comment:
not sure if restaurantId is needed when creating a new restaurant since the client wouldn't be on restaurantId page, but I could be wrong

Reply:
This route is creating a new menu for a given restaurant

Reply:
Yes, ignore my foolishness 👍